pkg/trace/agent/obfuscate.go

```diff
@@ -32,6 +32,7 @@ func (a *Agent) obfuscateSpan(span *pb.Span) {
 		for k, v := range span.Meta {
 			newV := o.ObfuscateCreditCardNumber(k, v)
 			if v != newV {
+				log.Debugf("obfuscating possible credit card under key %s from service %s", k, span.Service)
```

Comment:
So for normal tags that do contain CC numbers and should be obfuscated, wouldn't this happen very frequently? I don't know enough about log levels in the Agent, but maybe this is more of a `trace` level thing?

Reply:
That is certainly possible but I think is generally unlikely (and probably an instrumentation issue in that case). I went with debug here as it's by default off, but going to debug level is frequently a first step in investigating (and I believe going all the way to trace level is fairly rare).

Reply:
So are you saying that instrumentations shouldn't put things that are CC numbers into tags, and the CC obfuscation is just a fallback, and that is why this obfuscation doesn't happen often? Then I'm all good with `debug`.